rudder-lang/src/error.rs

```diff
@@ -205,3 +209,50 @@ impl fmt::Display for Error {
         }
     }
 }
+
+
+// Must have a trait to add the following method as ngrammatic::corpus only has an `add_text()` method
+trait FuzzyChecker<'src, I> {
+	fn add_vec(&mut self, list: I);
```

Comment:
you call it add_vec but you don't add a vector ?

Reply:
You have a generic parameter here but you only have a single implementation for an iterator